### PR TITLE
Added new REWARD tender type 

### PR DESCRIPTION
**Pull request by Liam-Watson**
Became needed as part of Vodacom telco rewards.

### CODE REVIEW

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 This page describes changes to the Service Interface Base API implemented across different releases of the interface.
+## 3.35.0
+Released 26th July 2021
+* Added new `REWARD` tender type for telco rewards.
```
Comment (BR00KE) [inline src/docs/devguide/hugo/site/content/release-notes.md:4]: We don't need to specify telco rewards.

**Review by BR00KE — CHANGES_REQUESTED** [review]
Other than small release notes change I think we can go straight to final version because it's such a small addition.

@Liam update pom version

File: src/main/java/io/electrum/vas/model/Tender.java [hunk 2]
```diff
@@ -62,6 +62,7 @@ public String toString() {
       ROUNDING("ROUNDING"),
       GIFT_CARD("GIFT_CARD"),
       LOYALTY_CARD("LOYALTY_CARD"),
+      REWARD("REWARD"),
```
Comment (simone-van-zyl) [inline src/main/java/io/electrum/vas/model/Tender.java:65]: I think we should be careful about changing the order of enum values 
image
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html
Rather add the new tendertype last in the list

**Review by simone-van-zyl — CHANGES_REQUESTED** [review]
Remember to update the Readme with the new version number

**Review by BR00KE — APPROVED** [review]

**Review by Shaaheen — APPROVED** [review]
LGTM

**Pull request merged**